js/ui/components/clubhouse.js

```diff
@@ -796,7 +797,9 @@ var ClubhouseComponent = new Lang.Class({
     Extends: SideComponent.SideComponent,
 
     _init: function() {
-        this._useClubhouse = this._imageUsesClubhouse() && this._hasClubhouse();
+        this._clubhouseApp = this._getClubhouseApp();
```

Comment:
Apps are already cached in `ShellAppSystem` and doing the lookup for the clubhouse app should be cheap (a hash table lookup) so I'd like to avoid storing it as an extra state here.